charts/dashboard/values.yaml

```diff
@@ -3,23 +3,22 @@
 
 image:
   repository: ghcr.io/cloudoperators/juno-app-greenhouse
-  tag: 0.3.2
+  digest: sha256:2c6c4c169a297239279cc159b07b504a581ece55df1295f87a0d5e12e6ed8a0f
```

Comment:
Will there be no more releases for `juno-app-greenhouse`?

Reply:
The image release process remains unchanged. I’m now using the image digest, which is included in the release, to clearly identify which version is actually deployed as latest.